app/domain/authentication/audit_event.rb

```diff
@@ -1,30 +1,39 @@
 # frozen_string_literal: true
 
 module Authentication
+
   AuditEvent = CommandClass.new(
     dependencies: {
-      get_role_by_login: GetRoleByLogin.new,
-      audit_log:         ::Authentication::AuditLog
+      role_cls: ::Role,
+      audit_log: ::Authentication::AuditLog
     },
-    inputs:       %i(input success message)
+    inputs: %i(input success message)
   ) do
 
     def call
       @audit_log.record_authn_event(
-        role:               role(@input.username, @input.account),
-        webservice_id:      @input.webservice.resource_id,
+        role: role,
+        webservice_id: @input.webservice.resource_id,
         authenticator_name: @input.authenticator_name,
-        success:            @success,
-        message:            @message
+        success: @success,
+        message: @message
       )
     end
 
     private
 
-    def role(username, account)
+    def role
       return nil if username.nil?
 
-      @get_role_by_login.(username: username, account: account)
+      @role_cls.by_login(username, account: account)
```

Comment:
Style nitpick: Because account is only using once, I think I'd delete the `account` private method and just do:
```
@role_cls.by_login(username, account: @input.account)
```
I think the `username` method is more defensible.   Again, this is a total nitpick.